runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -61,11 +70,23 @@
 	@Inject
 	private ItemManager itemManager;
 
+	@Inject
+	private KingdomConfig config;
```

Comment:
Not sure if I'm going crazy, or if every field and method has an extra indent.

Reply:
Looks to me like you're going crazy.

Reply:
On desktop everything had an extra indent except for decorators. On mobile it looks fine. I'm so confused.